### DISCUSSION TITLE
Asset dashboard

### DISCUSSION THREAD

**Discussion by Sahana-Nag** [snipe/snipe-it#9719] [Q&A]
Where to see the added custom fields in the Asset dashboard

**Comment by Bellwether** [comment]
Click the column customization icon and select which columns you want displayed.  


<img width="1353" alt="Image 2021-07-05 at 2 52 09 PM" src="https://user-images.githubusercontent.com/254543/124510111-2e3aa300-dda1-11eb-9fd7-34856d494981.png">
